apps/core/scenes/profile/Profile.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import * as React from 'react';
+import { ScrollView } from 'react-native';
+import { StyleSheet, Button, Card } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { margaritaTokens } from '@kiwicom/margarita-design-tokens';
+import { Illustration, Text } from '@kiwicom/margarita-components';
+
+type Props = {||};
+
+export default class Profile extends React.Component<Props> {
+  handleSignIn = () => {
+    // @TODO
+  };
+
+  render() {
+    return (
+      <ScrollView
+        style={styles.container}
+        contentContainerStyle={styles.containerContent}
+      >
+        <Illustration name="Lounge" style={styles.illustration} />
```

Comment:
Why is the illustration only on the web?

![Simulator Screen Shot - iPhone X - 2019-03-22 at 10 49 43](https://user-images.githubusercontent.com/858321/54814521-37ff7180-4c90-11e9-9c05-4faf39575251.png)

Reply:
I didn't included updated Illustration component in this PR. I just added it and should be fine now.

Also maybe if you are asking why I picked `Lounge` illustration instead of `Login` one. Simply because current login illustration available on server is outdated and quite ugly (with plane cut in half). There is already new way much better one prepared in design resources, so we can later swap it when it will be ready.